duffle.json

```
@@ -0,0 +1,50 @@
+{
+    "name": "riff",
+    "version": "0.1.0",
```

Replace with a default version of `0.0.0`, `0.1.0` is not a real version number for the bundle, but looks like one. Inject the actual version just before building. It would be nice it `duffle build` accepted a custom version.

```suggestion
    "version": "0.0.0",
```